tools/firmware.toit

```diff
@@ -122,10 +125,10 @@ write-file-or-print --path/string? output/string -> none:
     print output
 
 main arguments/List:
-  firmware-cmd := build-command
+  firmware-cmd := build-command --create-esp32-only
```

Comment:
So this way 'tools/firmware' keeps it old interface, but 'toit firmware' is updated?

Reply:
correct.